stdlib/runtime/numbers.gr

```diff
@@ -2489,3 +2489,58 @@ export let isBigInt = x => {
   let x = WasmI32.fromGrain(x)
   isBigInt(x)
 }
+
+// Scalbn is based on https://git.musl-libc.org/cgit/musl/tree/src/math/scalbn.c
+/*
+ * ====================================================
+ * Copyright (C) 2004 by Sun Microsystems, Inc. All rights reserved.
+ *
+ * Permission to use, copy, modify, and distribute this
+ * software is freely granted, provided that this notice
+ * is preserved.
+ * ====================================================
+ */
+/**
+ * Multiplies a floating-point number by an integral power of 2.
+ *
+ * @param x: The floating-point value
+ * @param n: The Integer exponent
+ * @returns The result of x * 2^n
+ *
+ * @since v0.5.4
+ */
+@unsafe
+export let scalbn = (x, n) => {
+  let (>) = WasmI32.gtS
+  let (<) = WasmI32.ltS
+  let (*) = WasmF64.mul
+  let (-) = WasmI32.sub
+  let (+) = WasmI32.add
+  let (<<) = WasmI64.shl
+  // Constants
+  let const_0x1p1023 = 8.98847e+307W
+  let const_0x1p_1022 = 2.22507e-308W
+  let const_0x1p53 = 9.0072e+15W
+  // Untag X and n
```

Comment:
```suggestion
  // Untag n and n
```

Reply:
Are these even being untagged?

Reply:
No They are not they were at one point and i missed removing the comment.